1_training_from_notebook/test_model.ipynb

```diff
@@ -91,12 +93,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.16"
+   "version": "3.11.2"
   },
   "orig_nbformat": 4,
   "vscode": {
    "interpreter": {
-    "hash": "63462a1f26ab486248b2a0fd058a0d9f9a6566a80083a3e1eb8f35617f2381b2"
+    "hash": "e7370f93d1d0cde622a1f8e1c04877d8463912d04d973331ad4851f04de6915a"
```

Comment:
I edited this in VScode. Not sure what those fields changed
Does it matter?

Reply:
Had no impact on the runtime, both the version and the display_name is just metadata for easily re-running on the same kernel.